Userland/Libraries/LibM/math.cpp

```diff
@@ -445,6 +445,11 @@ double tan(double angle) NOEXCEPT
     return ampsin(angle) / ampsin(M_PI_2 + angle);
 }
 
+float tanf(float angle) NOEXCEPT
+{
+    return (float)tan((double)angle);
```

Comment:
🤷🏻

Reply:
what can I say, it has the CxByte stamp of approval though! :smile:

Reply:
Are the manual casts really necessary? For me it was sufficient to write `return tan(angle);`

Reply:
not necessary, but at least this way it's very clear that there's casts both ways :)